lib/weather.html

```diff
@@ -1,7 +1,9 @@
 <div class="weather">
     <h3>Weather</h3>
+    <form action="">
```

Comment:
It is easier when you enter the city name to press enter to get the weather rather than to use the mouse and click on get weather button